Sources/XCHammer/XcodeTarget.swift

```diff
@@ -601,6 +602,31 @@ public class XcodeTarget: Hashable, Equatable {
                 break // These attrs are not related to XCConfigs
             case .binary:
                 break // Explicitly not handled since it is a implicit target we don't intend to handle
+            case .swift_language_version:
+                guard let swiftVersion = value as? String else {
+                    break
+                }
+                settings.swiftVersion <>=  First(swiftVersion)
+            case .has_swift_dependency, .has_swift_info:
+                guard let swiftVersion = try? ShellOut.shellOut(to: "xcrun swift package tools-version") else {
```

Comment:
This only works if there's a Package.swift, which not all projects will have. I've opened an issue: #124